### DISCUSSION TITLE
出海人去香港办银行卡最全攻略 | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#56] [General]
# 出海人去香港办银行卡最全攻略 | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/blogs/hk-bank-cards-guide

**Comment by jig68** [comment]
写的真好，详细很清晰，建议加一个每个银行年费更好。
